src/sentry/models/environment.py

```diff
@@ -20,6 +20,7 @@ class EnvironmentProject(Model):
 
     project = FlexibleForeignKey('sentry.Project')
     environment = FlexibleForeignKey('sentry.Environment')
+    is_hidden = models.BooleanField(default=False)
```

Comment:
There's ~250k some rows in this table, I can add this as a nullable boolean field instead if it's going to be be too big to add the column as not nullable with a default. I'd prefer this because it's simpler to maintain but I also wrote all of the existing code to handle if it had to be nullable (I think)

Reply:
I mean, the problem is that it's going to have to claim the lock needed, then lock the table for the duration of rewriting every row. I guarantee this will cause a blip of downtime.

We can make it NULL, backfill them all to False, then add NOT NULL.

Or just handle `None` as Falsey and don't rely on a strict `True` or `False` value? Your call, but we have to do one of these.